langkit/compile_context.py

```diff
@@ -1605,7 +1605,7 @@ def _compile(self, check_only=False, annotate_fields_types=False):
             PropertyPass('check overriding types',
                          PropertyDef.check_overriding_types),
             PropertyPass('check properties returning node types',
-                         PropertyDef.check_return_types),
+                         PropertyDef.check_returned_nodes),
```

Comment:
I think the previous name was better but whatevz

Reply:
Before your renaming the previous pass name was “check properties return types” (my intent was “check various things about the return types of properties”), so the method mirorred exactly the name of the pass. I thought your change called for a method rename as well, no?